service-base/src/main/java/org/eclipse/hono/service/registration/CompleteRegistrationService.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.service.registration;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.Verticle;
+import io.vertx.core.json.JsonObject;
+import org.eclipse.hono.util.RegistrationResult;
+
+/**
+ * A service for keeping record of device identities.
+ * 
+ * @see <a href="https://www.eclipse.org/hono/api/device-registration-api/">Device Registration API</a>
+ */
+public interface CompleteRegistrationService extends Verticle, RegistrationService {
```

Comment:
We don't need to extend `Verticle` here as it's already implied from the `RegistrationService`

Reply:
yep, that make sense